docs/docs/examples/embeddings/ipex_llm_gpu.ipynb

```diff
@@ -0,0 +1,112 @@
+    "\n",
+    "### For Windows Users with Intel Core Ultra integrated GPU\n",
+    "\n",
+    "In Anaconda Prompt:\n",
+    "\n",
+    "```cmd\n",
+    "set SYCL_CACHE_PERSISTENT=1\n",
+    "set BIGDL_LLM_XMX_DISABLED=1\n",
+    "```\n",
+    "\n",
+    "### For Linux Users with Intel Arc A-Series GPU\n",
+    "\n",
+    "```bash\n",
+    "# Configure oneAPI environment variables. Required step for APT or offline installed oneAPI.\n",
+    "# Skip this step for PIP-installed oneAPI since the environment has already been configured in LD_LIBRARY_PATH.\n",
+    "source /opt/intel/oneapi/setvars.sh\n",
+    "\n",
+    "# Recommended Environment Variables for optimal performance\n",
+    "export USE_XETLA=OFF\n",
+    "export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1\n",
+    "export SYCL_CACHE_PERSISTENT=1\n",
+    "```\n",
+    "\n",
+    "> **Note**\n",
+    ">\n",
+    "> For the first time that each model runs on Intel iGPU/Intel Arc A300-Series or Pro A60, it may take several minutes to compile.\n",
+    ">\n",
+    "> For other GPU type, please refer to [here](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#runtime-configuration) for Windows users, and  [here](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#id5) for Linux users.\n",
+    "\n",
+    "## `IpexLLMEmbedding`\n",
+    "\n",
+    "Setting `device=\"xpu\"` when initializing `IpexLLMEmbedding` will put the embedding model on Intel GPU and benefit from IPEX-LLM optimizations:\n",
+    "\n",
+    "```python\n",
+    "from llama_index.embeddings.ipex_llm import IpexLLMEmbedding\n",
+    "\n",
+    "embedding_model = IpexLLMEmbedding(\n",
+    "    model_name=\"BAAI/bge-large-en-v1.5\", device=\"xpu\"\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "> Please note that `IpexLLMEmbedding` currently only provides optimization for Hugging Face Bge models.\n",
+    "\n",
+    "You could then conduct the embedding tasks as normal:\n",
+    "\n",
+    "```python\n",
+    "sentence = \"IPEX-LLM is a PyTorch library for running LLM on Intel CPU and GPU (e.g., local PC with iGPU, discrete GPU such as Arc, Flex and Max) with very low latency.\"\n",
+    "query = \"What is IPEX-LLM?\"\n",
+    "\n",
+    "text_embedding = embedding_model.get_text_embedding(sentence)\n",
+    "print(f\"embedding[:10]: {text_embedding[:10]}\")\n",
+    "\n",
+    "text_embeddings = embedding_model.get_text_embedding_batch([sentence, query])\n",
+    "print(f\"text_embeddings[0][:10]: {text_embeddings[0][:10]}\")\n",
+    "print(f\"text_embeddings[1][:10]: {text_embeddings[1][:10]}\")\n",
+    "\n",
+    "query_embedding = embedding_model.get_query_embedding(query)\n",
+    "print(f\"query_embedding[:10]: {query_embedding[:10]}\")\n",
+    "```\n",
+    "\n",
+    "> **Note**\n",
+    ">\n",
+    "> You could refer to [here]() for a full example scripts of `IpexLLMEmbedding`."
```

Comment:
besides the link to the full example script, shall we add how to run the script command too? Or we just put everything in examples/README.md?